src/test/set1/set1ch7.test.ts

```diff
@@ -11,10 +11,10 @@ ava.test('decrypt aes 128 ecb with key', t => {
 })
 
 ava.test('encrypt aes 128 ecb with key', t => {
-    const data = fs.readFileSync(path.join(__dirname, '../../../files/set1ch6text2.txt'), { encoding: 'ascii' })
+    const data = 'EXACTLY 16 BYTES'
```

Comment:
After fixing the ECB method this test would fail because you were originally trying to encrypt/decrypt a 374 byte file (not a multiple of blocksize)